packages/cap-config-writer/src/cap-config/package-json.ts

```diff
@@ -119,3 +119,13 @@ function getWorkspacePackages(packageJson: Package): string[] | undefined {
 export function hasCdsPluginUi5(packageJson: Package): boolean {
     return !!packageJson.devDependencies?.['cds-plugin-ui5'];
 }
+
+/**
+ * Check if devDependency to '@sap/cds' is present in package.json.
+ *
+ * @param packageJson - the parsed package.json
+ * @returns true: devDependency to '@sap/cds' exists; false: devDependency to '@sap/cds' does not exist
+ */
+export function hasCdsPlugin(packageJson: Package): boolean {
+    return !!packageJson.devDependencies?.['@sap/cds'];
+}
```

Comment:
This function has the same name as the one above to redefines it!!! 
https://github.com/SAP/open-ux-tools/pull/2752/files#diff-55975eee6bdf69db481e1b03ab667532fcb86defc19dc1798ad810aa5242d02dR119
```suggestion
/**
 * Check if dependency to '@sap/cds' is present in package.json.
 *
 * @param packageJson - the parsed package.json
 * @returns true: dependency to '@sap/cds' exists; false: dependency to '@sap/cds' does not exist
 */
export function hasCdsModule(packageJson: Package): boolean {
    return !!packageJson.dependencies?.['@sap/cds'];
}
```